tests/src/Unit/AuthManagerTest.php

```diff
@@ -0,0 +1,116 @@
+<?php
+
+use Drupal\Tests\UnitTestCase;
+use PHPUnit\Framework\TestCase;
+//OAuth2Manager
+use Drupal\social_auth\AuthManager\OAuth2Manager;
+use Drupal\Core\Config\Config;
+use Drupal\Core\Logger\LoggerChannelFactoryInterface;
+use Drupal\social_api\AuthManager\OAuth2Manager as BaseOAuth2Manager;
+use Drupal\social_auth\AuthManager\OAuth2ManagerInterface;
+
+
+class AuthManagerTest extends UnitTestcase {
+
+  protected $settings;
+  protected $loggerFactory;
```

Comment:
Why do you keep declaring these variables as properties? If they are used only in one or two tests, they just need to be local variables to the methods. Declaring a variable as a class property gives the variable more meaning than just a box to store a value.

Reply:
Declared locally wherever required. These were supposed to be declared in the `setUp` method, but by mistake, I initialized them incorrectly. These are fixed now.